Cython/Utility/CpdefEnums.pyx

```diff
@@ -51,12 +51,14 @@ if PY_VERSION_HEX >= 0x03040000:
         ('{{item}}', {{item}}),
         {{endfor}}
     ]))
+    {{name}}.__doc__ = {{ repr(enum_doc) }}
```

Comment:
```suggestion
    if enum_doc is not None:
        {{name}}.__doc__ = {{ repr(enum_doc) }}
```

Reply:
Are you sure this is supposed to work? I'm not very comfortable with jinja, but when I tried to compile this locally I got `EnumType:53:7: undeclared name not builtin: enum_doc`.

So I guess you do want the keep the inherited python docstring when it's not set in cython? So I'll just remove the test testing the value of the docstring when it's not specified. Since I'm assuming we can't rely on python's enum docstring to not change!?

Reply:
Sorry, yes, you're totally right. Fixed my suggestion.

Reply:
> So I'll just remove the test testing the value of the docstring when it's not specified. Since I'm assuming we can't rely on python's enum docstring to not change!?

Yes. That's what I meant in my earlier comment with not testing the superclass docstring value itself.

Reply:
I removed the indentation because it didn't compile with it.